File: C++/Instructor.cpp
#include "Instructor.h"


	//constructor (initial value)
	Instructor::Instructor() : User(){
        
    }
	//Constructor (inputed values)
	Instructor::Instructor(string first_name, string last_name, string id) : User(first_name, last_name, id){

	}

	void Instructor::print_schedule(){
        cout << "Print Schedule for Instructor!" << endl;
    }
    void Instructor::print_classlist(){
        cout << "Print Classlist for Instructor!" << endl;
    }
    void Instructor::search_course(){
        cout << "Search course for Instructor!" << endl;
    }

    //Deconstructor
    Instructor::~Instructor(){

    }

File: Working/Instructor.py
from User import User

class Instructor(User):

    def __init__(self, id, first_name, last_name, title, hireyear, dept, email):
        super().__init__(id, first_name, last_name, email)
        self.title = title
        self.hireyear = hireyear
        self.dept = dept

    def show_info(self):
        print(self.id)
        print (self.first_name)
        print(self.last_name)
        print(self.title)
        print(self.hireyear)
        print(self.dept)
        print(self.email)

    def print_schedule(self):
        print("Print Schedule for Instructor!")
    def print_classlist(self):
        print("Print Classlist for Instructor!")
    def search_course(self):
        print("Search course for Instructor!")